centos-upgrade-first-stage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#
+# Script to run the first stage (everything through rebooting to flip into
+# the new kernel) for upgrading the CentOS AMIs from the imported version
+# to the latest in that major release series.
+#
+
+# Upgrade everything!
+sudo yum -y upgrade
+
+if test `cat /etc/centos-release | sed -n 's/CentOS release \([0-9]\).*/\1/p'` == 6; then
+    # Remove the 2.6.32 kernel series.
+    sudo yum -y remove kernel kernel-devel
+    export NEXT=`ls -1t /lib/modules/ | head -n 1`
+    echo "Guessing new kernel is $NEXT"
+    cd amzn-drivers-master/kernel/linux/ena
```

Comment:
I am missing on how we get this aws driver modules into centos machine, are we getting it from https://github.com/amzn/amzn-drivers/tree/master/kernel/linux/ena ? if so we are missing code on download.

Reply:
Yeah, this was me taking a shortcut; they're already on the image from the last update (shame on us for that).  I'll pull the latest version from Git.

Reply:
ok